packages/interface/src/features/ballot/components/SubmitBallotButton.tsx

```diff
@@ -44,36 +47,11 @@ export const SubmitBallotButton = ({ pollId }: ISubmitBallotButtonProps): JSX.El
     });
   }, [ballot, router, onVote, publishBallot, onVotingError, pollId, pollId]);
 
-  const handleOpenDialog = useCallback(() => {
-    setOpen(true);
-  }, [setOpen]);
-
   return (
-    <>
-      <Button variant={ableToSubmit ? "primary" : "disabled"} onClick={handleOpenDialog}>
-        {ableToSubmit ? "submit your ballot" : "Exceed initial voice credits"}
-      </Button>
-
-      <Dialog
-        button="primary"
-        buttonAction={handleSubmitBallot}
-        buttonName="submit"
-        description="This is not the final submission. Once you submit your ballot, you can change it during the voting period."
-        isLoading={isLoading}
-        isOpen={ableToSubmit && isOpen}
-        size="sm"
-        title="submit your ballot"
-        onOpenChange={setOpen}
-      />
+    <Button variant={ableToSubmit ? "primary" : "disabled"} onClick={handleSubmitBallot}>
+      {ableToSubmit && !isLoading ? "submit your ballot" : "Exceed initial voice credits"}
```

Comment:
Sometimes, the calculation is wrong and there is an issue about exceeding initial voice credits

<img width="1703" alt="image" src="https://github.com/user-attachments/assets/5c7f286e-e593-4de0-b5bc-144641041548">

Also seems like the first time the tx is triggered

<img width="1703" alt="image" src="https://github.com/user-attachments/assets/9e56e0c5-6b8c-4bab-98b8-b739429a8978">

Reply:
I just updated this PR, your first problem should be fixed, but I don't really understand your second question?

Reply:
Thanks. The second question was a bug, but seems like it has been solved